static/app/views/insights/browser/webVitals/components/webVitalDescription.tsx

```diff
@@ -111,17 +119,57 @@ export const VITAL_DESCRIPTIONS: Partial<
   },
 };
 
-export function WebVitalDetailHeader({score, value, webVital}: Props) {
+export function WebVitalDetailHeader({score, value, webVital, transaction}: Props) {
+  const hasSeerWebVitalsSuggestions = useHasSeerWebVitalsSuggestions();
   const status = score === undefined ? undefined : scoreToStatus(score);
 
+  const {mutate: createIssue} = useCreateIssue();
   return (
-    <div>
-      <WebVitalName>{`${WEB_VITAL_FULL_NAME_MAP[webVital]} (P75)`}</WebVitalName>
-      <WebVitalScore>
-        <Value>{value ?? ' \u2014 '}</Value>
-        {status && score && <PerformanceBadge score={score} />}
-      </WebVitalScore>
-    </div>
+    <SpaceBetweenFlex>
```

Comment:
If we're using the Flex component, we can just use the `justify` prop instead of styling the component